src/main/java/net/shrimpworks/unreal/archive/mirror/MirrorClient.java

```diff
@@ -22,41 +23,61 @@
  * progress).
  */
 public class MirrorClient implements Consumer<MirrorClient.Downloader> {
-
-	private final Deque<Content> content;
+	private final int retryLimit = 4;
```

Comment:
I think the `retryLimit`, being a fixed value, would be more clearly defined as `private static final int RETRY_LIMIT = 4`.

If at some later point, customisation via passing to the constructor is needed, it can be refactored accordingly.

Reply:
Yes, I thought about a --retry-limit, but I didn't want to change too much at once (also, if it fails 4 times, it will probably fail also a 5th).